client/src/notebooks/components/SessionCheatSheet.tsx

```diff
@@ -0,0 +1,153 @@
+    <code>{command}</code>
+    {
+      clipboard ? <Clipboard clipboardText={command} /> : null
+    }
+    <div className="renku-info" style={{ paddingTop: "3px" }}>{desc}</div>
+  </div>;
+}
+
+
+interface CommandsRowProps {
+  children: React.ReactNode;
+}
+function CommandsRow({ children }: CommandsRowProps) {
+  return <div className="d-flex justify-content-between flex-column flex-md-row commands-row flex-wrap">
+    {children}
+  </div>;
+}
+
+
+function Collaboration() {
+  const mergeDesc = <span>Incorporate the changes from the &lt;other branch&gt; into your branch.
+    See the <ExternalDocsLink url="https://www.atlassian.com/git/tutorials/using-branches/git-merge"
+    title="git merge docs" /> for details.
+  </span>;
+  return <div className="mb-3">
+    <CommandsRow>
+      <div>
+        <h2>Collaboration</h2>
+        <p className="renku-info">
+          Working with others requires coordination, and branching/merging is a common way to handle this.
+        </p>
+      </div>
+    </CommandsRow>
+    <CommandsRow>
+      <CommandDesc command="git checkout <branch>"
+        desc="Switch to the <branch>, replacing the contents of your project with the version in the branch." />
+      <CommandDesc command="git merge <other branch>"
+        desc={mergeDesc} />
+    </CommandsRow>
+  </div>;
+}
+
+
+function LearnMore() {
+  return <div className="mb-3">
+    <CommandsRow>
+      <div>
+        <h2>Want to learn more?</h2>
+      </div>
+    </CommandsRow>
+    <CommandsRow>
+      <div>
+        For a more detailed overview of common commands, see the {" "}
+        {/* eslint-disable-next-line max-len */}
+        <ExternalDocsLink url={`${RenkuPythonDocs.READ_THE_DOCS_ROOT}/_static/cheatsheet/cheatsheet.pdf`}
+          title="renku python cheat sheet"/>.
+      </div>
+      <div>
+        The <ExternalDocsLink url={`${RenkuPythonDocs.READ_THE_DOCS_ROOT}/reference/commands.html`}
+          title="Renku documentation" /> covers much more.
+      </div>
+    </CommandsRow>
+  </div>;
+}
+
+interface CommandsGroupProps {
+  group: GroupCommand;
+}
+function CommandsGroup({ group }: CommandsGroupProps) {
+  const commands = group.commands.map((command) => {
+    if (command.target.includes("ui")) {
+      return (<Col xs={12} md={6} lg={4} className="command" key={command.command}>
+        <CommandDesc command={command.command} desc={command.description}/></Col>);
+    }
+    return null;
+  });
+  return <div className="mb-3">
+    <CommandsRow>
+      <div><h2>{group.name}</h2></div>
+    </CommandsRow>
+    <CommandsRow>
+      {commands}
+    </CommandsRow>
+  </div>;
+}
+
+interface Command {
+  command: string;
+  description: string;
+  target: string[];
+}
+interface GroupCommand {
+  name: string;
+  commands: Command[];
+}
+function SessionCheatSheetGenerated() {
+  const [groups, setGroups] = useState<GroupCommand[]>([]);
+  useEffect(() => {
+    const data = JSON.parse(JSON.stringify(cheatsheetJson));
```

Comment:
We discussed being able to adapt the cheatsheet to different version of renku. I think it might make sense to make the the version a property.

Clients could do something like:

```
<SessionCheatSheetGenerated version={session.renkuVersion} />
```

Then, it wouldn't be necessary to use state in the component. It could be simplified to:

```suggestion
function SessionCheatSheetGenerated() {
  const groups = (version === 1) ? 
    cheatsheetJsonToGroups(cheatsheetJsonV1) : 
    cheatsheetJsonToGroups(cheetsheetJsonV2);
```